cosmic-core/test/integration/smoke/test_ip_exclusion_list.py

```diff
@@ -0,0 +1,343 @@
+import logging
+from marvin.cloudstackAPI import *
+from marvin.cloudstackTestCase import *
+from marvin.lib.base import *
+from marvin.lib.common import *
+from marvin.lib.utils import *
+from nose.plugins.attrib import attr
+
+class TestIpExclusionList(cloudstackTestCase):
+
+    attributes = {
+        'template_name': 'tiny linux kvm',
```

Comment:
Recently @remibergsma changed the tests not to have _hard-coded_ template names so the tests could work with XenServer too, for instance. Could you have a look at `cosmic-core/test/integration/smoke/test_public_ip_acl.py` as an example?